src/Cmf/MainBundle/Resources/data/posts/2015-04-01.yml

```diff
@@ -0,0 +1,62 @@
+name: "cmf-hackday-may-2015"
+parent: "/news"
+title: "CMF Hackday 1st/2nd May 2015"
+label: false
+format: markdown
+template: CmfMainBundle:Cms:news_detail.html.twig
+publish_start_date: "2015-04-01"
+body: |
+  We are happy to announce a joint Symfony CMF / Sulu.io hackday in Dornbirn on the 1st and 2nd of May. Join us
+  for a hacking, discussion and workshops (if there is interest).
+
+  If you would like to come, please register [here](<http register here dot com>)
```

Comment:
i was suggesting eventbrite before, but max was afraid of legal issues. google form sounds good to me, with the many options we have. less transparent for everybody, but that is ok in this case i think.